Derpitcraft/README.md

```diff
@@ -9,22 +9,22 @@ You can play Derpitcraft by registering for it: Use `/dcregister` in any channel
 ### Important details
 
 #### Version support
-Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.2**.
+Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.x**.
 
 #### How to join
-Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **DERPITCRAFT SEASON 9** NPC to join. You can join the SMP after **April 20th 2022**.
+Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **Derpitcraft Season 9** NPC to join. You can join the SMP after **April 20th 2022**.
 
 #### Spawn coordinates
 
-**spawn coordinates**: `[-161, -223]` 
+**Coordinates [X, Z]:**: `[-161, -223]` 
 
 This is world center. There is a world border at **7500 blocks**.
 
 #### Shopping District coordinates
 
-**shopping district center coordinates**: `[-250, -262]`
+**Coordinates [X, Z]:**: `[-250, -262]`
 
-Scroll down to read more about what the Shopping District is (**Pt.5**).
```

Comment:
They are points. If you want to make them parts, then make them all headings and make these links.